src/nix/store-check-trusted.cc

```diff
@@ -0,0 +1,61 @@
+#include <sstream>
+
+#include "command.hh"
+#include "logging.hh"
+#include "serve-protocol.hh"
+#include "shared.hh"
+#include "store-api.hh"
+#include "local-fs-store.hh"
+#include "util.hh"
+#include "worker-protocol.hh"
+
+using namespace nix;
+
+namespace {
+
+bool checkPass(const std::string & msg) {
+    if (isatty(fileno(stdin)))
+      notice(ANSI_GREEN "[PASS] " ANSI_NORMAL + msg);
+    return true;
+}
+
+bool checkFail(const std::string & msg) {
+    if (isatty(fileno(stdin)))
+      notice(ANSI_RED "[FAIL] " ANSI_NORMAL + msg);
+    return false;
+}
```

Comment:
I don't think these colors bring a lot here. This is low-level plumbing, we should probably stick to the most sober solution

Reply:
Then we should be removing that in another PR, for `nix doctor` itself. But for this PR to remain a simple addition, I'm not going to touch the existing code/pattern that doctor.cc implements.

Reply:
No, I think that's fine for `doctor` which is intended as a higher-level “check for obvious configuration issues” command. I just think we don't need it in this command

Reply:
Ah, I misread the source file name, I thought the code highlighted was from doctor.cc

In this case, it's not displayed unless your input `isatty`. This means it handles the case of scripting where you don't want any stdout, only an exit code, as well as also telling the user whether they passed or failed trust checking.

It is only in the interactive case where the input is a tty, where you would want any output from the command at all. So what would you rather it print? The same text without the pass/fail context text prepended?

Reply:
After some thinking, I think I agree now. I was just copying Nix doctor and liked the pretty colors. But it's not sober, as you mention.

Reply:
I've removed the color and fail/pass text.